src/realm/alloc_slab.cpp

```diff
@@ -110,7 +111,7 @@ inline SlabAlloc::Slab::Slab(ref_type r, size_t s)
     , size(s)
 {
     total_slab_allocated.fetch_add(s, std::memory_order_relaxed);
-    addr = static_cast<char*>(util::mmap_anon(size));
+    addr = new char[size];
```

Comment:
I think this needs some kind of forced alignment. In principle new could return an unaligned pointer.

Reply:
We are ensured 16 bit alignment